### ISSUE TITLE
Ghost Tag page and Author page lack proper social meta

### ISSUE THREAD

**Issue by ahmadabdolsaheb** [freeCodeCamp/freeCodeCamp#39573]
<img width="605" alt="Screen Shot 2020-09-14 at 7 23 04 AM" src="https://user-images.githubusercontent.com/4591597/93046763-51099880-f663-11ea-8448-c0bb35bf2c31.png">

**Comment by scissorsneedfoodtoo** [comment]
Played around with this on .dev/news and it seems like all we need to do is set the default Twitter and Facebook card images in the Ghost dashboard:

![image](https://user-images.githubusercontent.com/2051070/93057468-eda25980-f6a8-11ea-9621-c7ab654d5529.png)

There's a way to set a different image for each tag, but it may be enough to use the basic publication cover image for now.

**Comment by scissorsneedfoodtoo** [comment]
Will run this by @QuincyLarson real quick before changing those settings.

**Comment by scissorsneedfoodtoo** [comment]
Did a bit more testing and it seems like author pages will show the author's cover image when shared. We'll need to find a good way to add a default image in the theme.

For the tags pages, I went ahead and added the default cover image for both Twitter and Facebook. I also started adding custom images for the most popular tags like tech, programming, technology, and so on. I'll continue to do that for the popular tags and programming languages.
